minecraft_servers/gazami/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+    "server_name": "gazami",
+    "nice_name": "Gazami",
+    "direct_ip": "gazami.de",
+    "server_wildcards": [
+        "gazami.de",
+        "gazami.eu"
+    ],
+    "supported_languages": [
+        "de"
+    ],
+    "social": {
+        "web": "https://gazami.de",
+        "threads": "gazami.de",

```

Comment:
`social.threads` is also not a valid prop. See https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#available-keys

Reply:
Changed

Reply:
Now its changed